src/components/PinItButton.js

```diff
@@ -80,15 +83,21 @@ export default class PinItButton extends PinterestBase {
      * @returns {JSX}
      */
     renderPinOne() {
-        const {pin, media, url, description} = this.props;
+        const {pin, media} = this.props;
+        let {url, description} = this.props;
         let href;
         if (pin) {
             href = Const.URL.REPIN.replace('<id>', pin) + `?guid=${Config.guid}`;
         } else {
+            if(isBrowser) {
+              url = url || window.document.URL;
+              description = description || window.document.title;
+            }
+
             href = Const.URL.PIN_CREATE + `?guid=${Config.guid}`;
             href += `&media=${encodeURIComponent(media)}`;
-            href += `&url=${encodeURIComponent(url || document.URL)}`;
-            href += `&description=${encodeURIComponent(description || document.title)}`;
+            href += `&url=${encodeURIComponent(url)}`;
+            href += `&description=${encodeURIComponent(description)}`;
```

Comment:
Oh shoot, I did not even pay attention, this is not a click-handler anyway. Yup. Good call.